pyproject.toml

```diff
@@ -64,6 +64,8 @@ mkdocs-redirects = ">=1.2.1"
 pythonpath = "."
 addopts = "--cov --cov-report=html --cov-report=xml --junitxml=out/test_results/junit.xml"
 
+
+
```

Comment:
Not needed

Reply:
some auto formatting from my IDE, fixed. :)